examples/index.js

```diff
@@ -100,6 +108,63 @@ class CustomArrows extends Component {
   }
 }
 
+class ControlPlay extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { playing: false };
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ playing: true });
+    }, 5000);
+  }
```

Comment:
@xseignard Can you clear the timeout in componentDidUnmount Otherwise there will be a lot of errors, if the component is unmounted?